busbud/coding-challenge-backend-c
# Adding a suggestion scoring for loactions autocomplete

## Description
a design for an API endpoint that provides autocomplete suggestions for large cities.
The suggestions should be restricted to cities in the USA and Canada with a population above 5000 people


## Dependencies
None

## How to test?

- run npm test
- - all tests should pass 
- run this endpoint from the server with your query (https://busbud-suggestion.herokuapp.com/suggestions)

## Screenshots:
NA
